### PR TITLE
LPS 113033

### PR DESCRIPTION
**Pull request by diegonvs**
After some internal conversations on https://issues.liferay.com/browse/LPS-113033, there aren't any product team using this AUI component. All the occurrences of this component were replaced with [`liferay-ratings:ratings`](https://github.com/liferay/liferay-portal/tree/bc2244a90ba221bdcac72de9d58e5e8afcba88c7/modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js) Taglib that is  currently implemented by React.

I'm currently adding something on https://github.com/liferay/liferay-portal/blob/master/readme/BREAKING_CHANGES.markdown file

/cc @jbalsas @ambrinchaudhary

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:test:sf

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/ratings.js [hunk 1]
```diff
@@ -12,734 +12,9 @@
...
-	},
-	'',
-	{
-		requires: ['aui-rating'],
-	}
-);
+/**
+ * @deprecated As of Athanasius (7.3.x), with no direct replacement
+ */
+AUI.add('liferay-ratings', () => {}, '', {
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/ratings.js]: Just the deprecation notice is enough. Nuking the module is not necessary!

**Comment by ambrinchaudhary** [comment]
Hey @diegonvs, this changes means that the type "stacked-stars" will no longer work, as it has logic related to it, is it right? Maybe we cannot deprecate it yet...

**Comment by jbalsas** [comment]
Hey @diegonvs, left you a comment in there... if you don't nuke the module implementation (which you shouldn't do...), there's no need to add a Breaking Change, since we're simply marking this module as deprecated.

Please, address that and send to @wincent for final review! 😘

**Comment by jbalsas** [comment]
> Maybe we cannot deprecate it yet...

Deprecation doesn't mean TOTAL DESTRUCTION OF THE HEART 😂

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/ratings.js (hunk 1 above)
Comment (diegonvs) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/ratings.js]: Ok!

**Pull request closed without merging**